esx_upgraded_duty/client/main.lua

```diff
@@ -113,18 +113,34 @@ Citizen.CreateThread(function()
 							local r,g,b = 0,255,0
 							local duty = _U('duty1')
 							if playerjob == v.offjob then duty = _U('duty2') r,g,b = 255,0,0 end
-							DrawMarker(6, v.Pos.x, v.Pos.y, v.Pos.z - 0.975, 0.0, 0.0, 0.0, -90.0, 0.0, 0.0, v.Size.x, v.Size.y, v.Size.z, r,g,b, 100, false, true, 2, false, false, false, false)
+							--DrawMarker(6, v.Pos.x, v.Pos.y, v.Pos.z - 0.975, 0.0, 0.0, 0.0, -90.0, 0.0, 0.0, v.Size.x, v.Size.y, v.Size.z, r,g,b, 100, false, true, 2, false, false, false, false)
```

Comment:
make a new value in config called config.drawmarker and check if its true then draw marker don't hardcode